src/core/AutomatableModel.cpp

```diff
@@ -638,23 +641,28 @@ ValueBuffer * AutomatableModel::valueBuffer()
 			return &m_valueBuffer;
 		}
 	}
-	AutomatableModel* lm = NULL;
-	if( hasLinkedModels() )
-	{
-		lm = m_linkedModels.first();
-	}
-	if( lm && lm->controllerConnection() && lm->controllerConnection()->getController()->isSampleExact() )
+
+	if (!m_controllerConnection)
```

Comment:
I think that conditional could be removed. If this model has a controller, but the value of the controller wasn't used (`useControllerValue == false`, meaning previous conditional was skipped), shouldn't it check for the linked models?

Reply:
If useControllerValue == false it means that it's playing an automation pattern so it shouldn't check the linked models